src/modules/sync/index.tsx

```diff
@@ -34,13 +42,13 @@ function SiteSyncDescription( { children }: PropsWithChildren ) {
 				</div>
 				<div className="max-w-[40ch] text-a8c-gray-70 a8c-body">
 					{ __(
-						'Connect your existing WordPress.com or Pressable sites with Jetpack activated, or create a new one. Then share your work with the world.'
+						'Launch your existing WordPress.com or Jetpack-activated Pressable sites, or import an exisiting one. Then, share your work with the world.'
```

Comment:
There is a typo here:

```suggestion
						'Launch your existing WordPress.com or Jetpack-activated Pressable sites, or import an existing one. Then, share your work with the world.'
```

Reply:
Well spotted! Fixed in [bb66758](https://github.com/Automattic/studio/pull/2002/commits/bb6675808d6b5569851152d037574f93002b4e80)